resolver/request.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+import requests
+from validator_collection import checkers
+
+from sceptre.resolvers import Resolver
+
+
+def make_request(url):
+    """
+    Make a request to a REST API endpoint
+    :param url: The url endpoint reference
+    """
+    content = None
+    try:
+        response = requests.get(url)
+        content = response.text
+        if response.status_code != requests.codes.ok:
+            raise response.raise_for_status()
```

Comment:
You should invoke `raise_for_status()` all the time. It does nothing if it was a 200 request.

Reply:
done